server/channels/connection.js

```diff
@@ -0,0 +1,33 @@
+const emit = require('../utils').emit;
+const logger = require('../logging');
+
+const getActiveGenfors = require('../models/meeting').getActiveGenfors;
+const getActiveQuestion = require('../models/issue').getActiveQuestion;
+
+const connection = (socket) => {
+  getActiveGenfors().then((meeting) => {
+    if (!meeting) {
+      emit(socket, 'OPEN_MEETING', { error: 1, code: 'no_active_meeting', message: 'Ingen aktiv generalforsamling.' });
```

Comment:
Do we need to say that this is an error? I'd say an error is more like a submit action failing. This seems more like a certain kind of state to me than an actual error.